### ISSUE TITLE
Rust Visual C++ prerequisites: Cancel

### ISSUE THREAD

**Issue by Adarma** [rust-lang/rustup#3088]
### Problem you are trying to solve

I'm setting up a new machine and will install rust.
I will also install Microsoft Visual Studio.

I started to install rust first...
Rustup gives 3 options to install Visual Studio components, but I just want to quit rustup, install Microsoft Visual Studio myself and run rustup again afterwards.
There is no option to quit rustup.
```
Rust Visual C++ prerequisites

Rust requires a linker and Windows API libraries but they don't seem to be
available.

These components can be acquired through a Visual Studio installer.

1) Quick install via the Visual Studio Community installer
   (free for individuals, academic uses, and open source).

2) Manually install the prerequisites
   (for enterprise and advanced users).

3) Don't install the prerequisites
   (if you're targetting the GNU ABI).
```

### Solution you'd like

There should also be:
```
4) Quit install
```

**Comment by Diggsey** [comment]
You can just Ctrl+C or close the window, but there's no harm in having the option either.

**Comment by Adarma** [comment]
Yes, but it is not clear that force quitting will not leave an incomplete install mess.
Maybe nothing yet happened, maybe it has...

**Comment by ChrisDenton** [comment]
Hm, selecting either option 2 or 3 will then give an option to quit the installer without making any changes (and even with option 1 you have the opportunity to cancel).

The option you wanted in this case is 2 ("quit rustup, install Microsoft Visual Studio myself and run rustup again"). Perhaps this could be made clearer in some way.
